modules/http4s-munit/src/main/scala/munit/Http4sSuite.scala

```diff
@@ -105,17 +115,25 @@ abstract class Http4sSuite[A: Show] extends CatsEffectSuite {
       if (maxParallel < 1) Assertions.fail("maxParallel must be > 0")
       else copy(maxParallel = maxParallel.some)
 
-    /** Force the test to be executed just once */
-    def doNotRepeat = copy(repetitions = None)
-
```

Comment:
I think we should keep the `doNotRepeat` method as a simple alias `def doNotRepeat = repeat(1)`. It is easier to read and understand than `repeat(1)`

Reply:
See [`3e8d5ba` (#42)](https://github.com/alejandrohdezma/http4s-munit/pull/42/commits/3e8d5ba1434a85b06ae90761c5f7d4cc9f6868a0)